Projects/App/Sources/AppDelegate+Tracking.swift

```diff
@@ -34,7 +34,9 @@ extension AppDelegate {
         let store: ProfileStore = globalPresentableStoreContainer.get()
         if let userId = store.state.memberDetails?.id {
             let analyticsService: AnalyticsClient = Dependencies.shared.resolve()
-            analyticsService.setWith(userId: userId)
+            Task {
+                try await analyticsService.setWith(userId: userId)
```

Comment:
setWith should be async. It doesn't do anything that would block existing thread. Can we remove async for setWith and just call it like we used to?